### ISSUE TITLE
Weird behavior when typing values in Windows Filters LV2.

### ISSUE THREAD

**Issue by Lowclock** [lsp-plugins/lsp-plugins#388]
[Here's an example of where I entered 20k with the keyboard into the frequency control](https://imgur.com/zoFf6OI) Neither the curve it shows, or the frequency response it outputs, actually corresponds to the number entered. The audible response isn't even the erroneous curve either, just some other weird higher value that also only plays out of one speaker that varies between L and R sometimes. Using the mouse to control any of the filter properties will cause it to correctly snap to whatever value it should be. Doesn't happen on Linux, or with any other plugins I tried.

GNU Make 4.4
MinGW-W64 8.1.0
PHP 8.3.3
Carla 2.5.8
LSP 1.2.16-Devel
Windows 10 Pro, Ryzen 3700x

**Comment by sadko4u** [comment]
Can not reproduce the problem on 32-bit Windows nor on 64-bit Windows. I just type '20000' into the frequency control and all changes well.

**Comment by Lowclock** [comment]
[It's very consistent and measurable for me.](https://imgur.com/lVpXqdQ) Even hitting the reset button doesn't actually snap that curve or response back to 10khz. It always gets stuck at some weird value in between the previous and new value. Is there any way I could get you more info that would be helpful?

**Comment by sadko4u** [comment]
I need a stable case to reproduce the problem. This also can be related to the host. Do you use Carla with JACK on Windows?

**Comment by Lowclock** [comment]
Nope. I use it on Linux but I honestly didn't even know it was a thing on Windows. I use FlexASIO on Windows.
